server/routes/documents/failed-to-attend/failed-to-attend.controller.js

```diff
@@ -0,0 +1,153 @@
+            juror.id = i;
+          });
+        } else {
+          response = await reissueLetterDAO.getList(app, req, payload);
+        }
+
+        req.session.documentsJurorsList = response;
+        app.logger.info('Fetched the list of juror documents: ', {
+          auth: req.session.authentication,
+          jwt: req.session.authToken,
+          data: { ...payload },
+        });
+        return res.redirect(urljoin(app.namedRoutes.build('documents.letters-list.get', {
+          document: document,
+        }), urlBuilder(req.body)));
+      } catch (err) {
+        // A 404 means no results were found
+        if (err.statusCode === 404) {
+          req.session.documentsJurorsList = {
+            headings: [],
+            'data_types': [],
+            data: [],
+          };
+
+          return res.redirect(urljoin(app.namedRoutes.build('documents.letters-list.get', {
+            document,
+          }), urlBuilder(req.body)));
+        }
+
+        app.logger.crit('Failed to fetch documents / jurors list: ', {
+          auth: req.session.authentication,
+          jwt: req.session.authToken,
+          data: { ...req.body },
+          error: (typeof err.error !== 'undefined') ? err.error : err.toString(),
+        });
+
+        return res.render('_errors/generic.njk');
+      }
+    };
+  };
+
+  module.exports.getSingleFailedToAttend = function(app) {
+    return async function(req, res) {
+      const { document } = req.params;
+      const { jurorNumber } = req.query;
+      const { singleFTAdate } = req.query;
+
+      delete req.session.errors;
+
+      if (jurorNumber) {
+
+        req.session.documentsJurorsList ={
+          checkedJurors: [{ 'juror_number': jurorNumber }],
+          data : [{'juror_number': jurorNumber, 'absent_date': singleFTAdate}],
+        };
+        req.session.bannerMessage = 'Document sent for printing';
+
+        return res.render('custom-components/letter-flow/redirect.njk', {
+          letterType: 'failed-to-attend',
+          completeRoute: app.namedRoutes.build('juror-record.attendance.get', {
+            jurorNumber,
+          }),
+          // eslint-disable-next-line max-len
+        });
+      }
+
+      try {
+
+
+      } catch (err) {
+        // A 404 means no results were found
+        if (err.statusCode === 404) {
+          req.session.documentsJurorsList = {
+            headings: [],
+            'data_types': [],
+            data: [],
+          };
+
+          return res.redirect(urljoin(app.namedRoutes.build('documents.letters-list.get', {
+            document,
+          })));
+        }
+
+        app.logger.crit('Failed to fetch documents / jurors list: ', {
+          auth: req.session.authentication,
+          jwt: req.session.authToken,
+          data: { ...req.body },
+          error: (typeof err.error !== 'undefined') ? err.error : err.toString(),
+        });
+
+        return res.render('_errors/generic.njk');
+      }
+
+    };
```

Comment:
is any of this necessary with a en empty `try{}` block?

Reply:
Removed